HIP/hip-1086.md

```diff
@@ -89,6 +89,9 @@ The pricing will be determined by the usual gas price logic in the EVM world.
 This HIP only permits the `EthereumTransaction` to be "jumbo" transactions. Any other type of transactions exceeding 6KB should 
 be rejected unless specifically allowed by a future HIP.
 
+This HIP only applies to single transactions. Jumbo Ethereumtransactions will not be allowed to be executed on Batch transactions,  
+as the 6kb still applies to the batching functionality.
```

Comment:
We can word it out instead of a number in case Hiero ever changes the limit.
```suggestion
as the standard transaction size limit still applies to the batching functionality.
```